### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by MamunC0der**
### File: `op-supervisor/README.md`
1. Fixed typo in word:
   - Old: `desgin-doc`
   - New: `design-doc`

### File: `op-proposer/README.md`
2. Fixed typo in word:
   - Old: `managent`
   - New: `management`

### CODE REVIEW

File: op-supervisor/README.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ Full support for chain reorgs (detecting them, and resolving them) is the
 next priority after the above architecture and data changes.
 
 Further background on the design-choices of op-supervisor can be found in the
-[superchain backend desgin-doc](https://github.com/ethereum-optimism/design-docs/blob/main/protocol/superchain-backend.md).
+[superchain backend design-do](https://github.com/ethereum-optimism/design-docs/blob/main/protocol/superchain-backend.md).
```
Comment (smartcontracts) [inline op-supervisor/README.md:274]: Original was correct here

Comment (smartcontracts) [inline op-supervisor/README.md]: Should be `design doc`

Comment (protolambda) [inline op-supervisor/README.md]:
```suggestion
[superchain backend design-doc](https://github.com/ethereum-optimism/design-docs/blob/main/protocol/superchain-backend.md).
```

**Review by protolambda — APPROVED** [review]

**Review by protolambda — APPROVED** [review]

**Pull request merged**